hack/boilerplate.go.txt

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright 2019 Wrangler Sample Controller Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
I ended up moving this to the codegen directory in eks-operator since hack isn't a super informative name. I recommend doing the same. Be sure to update the reference to this in `codegen/main.go`